I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 39, in script
    scene justin marlyn BG
Exception: Image 'justin' does not accept attributes 'marlyn BG'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 39, in script
    scene justin marlyn BG
  File "C:\Users\msmad\Downloads\Waterloo Games - Renpy\renpy-8.3.3-sdk\renpy\ast.py", line 1245, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\msmad\Downloads\Waterloo Games - Renpy\renpy-8.3.3-sdk\renpy\ast.py", line 1074, in show_imspec
    renpy.config.show(name,
  File "C:\Users\msmad\Downloads\Waterloo Games - Renpy\renpy-8.3.3-sdk\renpy\exports\displayexports.py", line 477, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\msmad\Downloads\Waterloo Games - Renpy\renpy-8.3.3-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\msmad\Downloads\Waterloo Games - Renpy\renpy-8.3.3-sdk\renpy\display\displayable.py", line 249, in _duplicate
    args.extraneous()
  File "C:\Users\msmad\Downloads\Waterloo Games - Renpy\renpy-8.3.3-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'justin' does not accept attributes 'marlyn BG'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.3.24111502
theWaterlooGames 1.0
Wed Nov 20 17:53:34 2024
